ComputerVisionServer/moedx/client/multi_client.py

```diff
@@ -74,10 +74,14 @@ class Client:
     stats_latency_full_process = RunningStats()
     stats_latency_network_only = RunningStats()
     stats_server_processing_time = RunningStats()
-    stats_cpu_utilization = RunningStats()
-    stats_mem_utilization = RunningStats()
-    stats_gpu_utilization = RunningStats()
-    stats_gpu_mem_utilization = RunningStats()
+    stats_cpu_util = RunningStats()
```

Comment:
Not immediately clear from the pull what it is: These all use RunningStats() to init?